cmd/minishift/cmd/addon/addon_install.go

```diff
@@ -44,7 +44,7 @@ var (
 var addonsInstallCmd = &cobra.Command{
 	Use:   "install [SOURCE]",
 	Short: "Installs the specified add-on.",
-	Long:  "Installs the add-on from the specified file path and verifies the installation.",
+	Long:  "Installs the specified add-on from the specified file path and verifies the installation.",
```

Comment:
why "specified" add-on? You are not specifying and add-on, you are specifying an installation source in form of a path.